src/config/quizConfig.ts

```diff
@@ -0,0 +1,55 @@
+import dogImg from '@/assets/dog.png';
+import bunnyImg from '@/assets/bunny.png';
+import catImg from '@/assets/cat.png';
+
+export interface Answer {
```

Comment:
You can add the "isCorrect: boolean" attribute in here, so you dont need the "correctAnswer" attribute in the Quiz Type and you only need to pass the "Answer" into your QuizAnswer component